### ISSUE TITLE
Fix KDE Apps on GNOME

### ISSUE THREAD

**Issue by trytomakeyouprivate** [iaacornus/Fedora-OSTree-Setup#96]
So I was brave and tried Silverblue. It is pretty horrible, as I think Dolphin, Okular, Kate and many other apps beat Gnome apps by far.

And in general, a DE should support all apps by default.

So to fix these apps, what would you do? This focuses on Flatpaks, so I guess installing `plasma-systemsettings` would not work?

Normally I installed the Kvantum theme flatpak, I dont know how that works, if it maybe gets the native Qt theme and adapts. 

there is also `qt-qtconfig` (QT4) or `qt3-config` (QT3) that may be enough and dont have so many bundled apps with them. I will try these out.

**Comment by trytomakeyouprivate** [comment]
installing systemsettings made Nautilus show KDE icons, weird

**Comment by iaacornus** [comment]
i agree on that, okular is far better than evince, while dolphin is more functional than gnome's default. although the problem is the inconsistency, hence the problems such as in color scheme, icons and scales

**Comment by trytomakeyouprivate** [comment]
I tried, and its incredibly hard. Maybe we can get some help, I gave up currently.
